frontends/mit-open/src/page-components/SearchDisplay/ResourceCategoryTabs.tsx

```diff
@@ -8,16 +8,30 @@ import {
 } from "ol-components"
 import { ResourceCategoryEnum, LearningResourcesSearchResponse } from "api"
 
-const TabsList = styled(TabButtonList)({
+const TabsList = styled(TabButtonList)(({ theme }) => ({
   ".MuiTabScrollButton-root.Mui-disabled": {
     display: "none",
   },
-})
+  [theme.breakpoints.up("md")]: {
+    "div div button:first-of-type": {
+      minWidth: "85px",
+    },
+    "div div button:nth-of-type(2)": {
+      minWidth: "112px",
+    },
+    "div div button:nth-of-type(3)": {
+      minWidth: "118px",
+    },
+    "div div button:last-of-type": {
+      minWidth: "172px",
+    },
+  },
+}))
 
 const CountSpan = styled.span`
-  min-width: 35px;
-  text-align: left;
+  font-weight: ${({ theme }) => theme.typography.fontWeightRegular};
```

Comment:
**Requested Change:** Use a named style. In this case, it should be `typography.body3`.
```
const CountSpan = styled.span(({ theme }) => ({
  ...theme.typography.body3,
}))
```

Here's where you can see it in Figma (though until yesterday i didn't realize a single chunk of text in figma could have multiple styles like this.)

![Screenshot 2024-07-24 at 11 00 15 AM](https://github.com/user-attachments/assets/ce17dcc4-ae2d-4ea2-b531-27614cda0676)